src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Services/EnterpriseUserAuthenticationService.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Net.Http;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Text.Json;
+using Altinn.Platform.Authentication.Model;
+using Altinn.Platform.Authentication.Services.Interfaces;
+
+namespace Altinn.Platform.Authentication.Services
+{
+    /// <inheritdoc/>
+    public class EnterpriseUserAuthenticationService : IEnterpriseUserAuthenticationService
+    {
+        /// <summary>
+        /// Method for authenticating enterpriseuser at the SBLbridge
+        /// </summary>
+        /// <param name="credentials">Credentials of the enterpriseuser</param>
+        /// <param name="bridgeApiEndpoint">SBLbridge-endpoint</param>
+        public ConfiguredTaskAwaitable<HttpResponseMessage> GetResponseMessage(EnterpriseUserCredentials credentials, string bridgeApiEndpoint)
+        {
+            HttpClient client = new HttpClient();
```

Comment:
This will likely cause socket exhaustion in high load scenarios, as the client is disposed upon exiting the method but the underlying socket is still claimed. HttpClient instances are thread-safe, and should be reused. See Startup.cs on how you can use `services.AddHttpClient` in to let the service utilize a named HttpClient that will be properly reused.

Reply:
Added in latest commit.